source/partials/release_notes/_release-19-6-0.md.erb

```diff
@@ -1,3 +1,7 @@
+<p class="information">
+  This release has a known issue <%= link_to_issue '6666', '' %> which does not automatically start the GoCD server/agent on restart of the server. If this can affect your setup we would recommend upgrading directly to GoCD 19.7.0 which fixes this issue.
```

Comment:
@maheshp -- as we've added a warning on 19.6.0 release notes, do we still need these changes?